web/packages/teleport/src/Discover/ConnectMyComputer/SetupConnect/SetupConnect.story.tsx

```diff
@@ -105,11 +105,11 @@ export const PollingSuccess = () => (
 PollingSuccess.parameters = {
   msw: {
     handlers: [
-      rest.get(cfg.api.nodesPath, (req, res, ctx) => {
-        return res.once(ctx.json({ items: [] }));
+      http.get(cfg.api.nodesPath, () => {
+        return HttpResponse.json({ items: [] });
```

Comment:
it doesn't work because the request itself is broken. they are returning 404s so its like... msw is and isn't intercepting all at the same time.

For some reason, none of my `msw` requests are working now (seems like only `enterprise` urls are working). I'm not sure what changed but I'll look into it. 

It was working during my fix so it must be related to something else. I'll keep digging around

Reply:
Ok so, if msw fails to fetch a URL (like it does everytime i go to polling success), then it breaks for everything else and nothing will work.

I've been fiddling with it for a few hours and still not sure the cause but this is a great story to look at as its the one that breaks.

Reply:
When debugging this story, I found that the failing requests that msw complains about belong to some completely different stories. This made me realize that we have multiple stories which manually provide `mswLoader` and call `initialize`. I believe this has to be removed before we venture any further. It might even resolve the issue itself, as otherwise I don't see any problems with the config (I was testing things with Grzegorz's patch applied).

Reply:
@ravicious can you check again? 

i noticed that some components who calls api from `teleportContext.<someService>.<apiCall>`, using the msw doesn't work, and i'm not sure why.


> This made me realize that we have multiple stories which manually provide mswLoader and call initialize

also i wasn't sure how i can tell this 🤔

Reply:
> > This made me realize that we have multiple stories which manually provide mswLoader and call initialize
> 
> also i wasn't sure how i can tell this 🤔

Michael already remove that from the stories, so there shouldn't be any that do this. Here's [an example of how this looked like](https://github.com/gravitational/teleport/pull/44041/commits/9122c178de28e38ce496ce7cd116edfc9931124e#diff-a6b09f9c6490139b7055114887d35ae202681837b28899d592700c426ce31134).